docs/how_to_add_packages.md

```diff
@@ -1,8 +1,11 @@
 # Adding Packages to ConanCenter
 
-The [conan-center-index](https://github.com/conan-io/conan-center-index) (this repository) contains recipes for the remote [JFrog ConanCenter](https://conan.io/center/).
-This remote is added by default to a clean installation of the Conan client. Recipes are contributed by opening pull requests as explained in the sections below.
-When pull requests are merged, the CI will upload the generated packages to the [conancenter](https://conan.io/center/) remote.
+ConanCenterIndex aims to provide the best quality packages for any open source project.
```

Comment:
```suggestion
ConanCenterIndex aims to provide the best quality packages of any open source project.
```

Can't tell if this is the best quality package of open source projects or should it be for any project, i.e. closed or open source?